.infrastructure/app/deployment.yml

```
@@ -90,4 +90,24 @@ spec:
           secretName: app-secret
       - name: app-config-volume
         configMap:
-          name: app-config
\ No newline at end of file
+          name: app-config
+      affinity:
+        nodeAffinity:
+          PreferedDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
+                  - key: "app"
+                    operator: In
```

Need to set weight and preference. Find more info in [doc](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity)
```suggestion
            - weight: 1
              preference:
                 matchExpressions:
                  - key: "app"
                    operator: In
```